Formula/jpeg-xl.rb

```diff
@@ -41,26 +43,22 @@ class JpegXl < Formula
 
   # These resources are versioned according to the script supplied with jpeg-xl to download the dependencies:
   # https://github.com/libjxl/libjxl/tree/v#{version}/third_party
-  resource "lodepng" do
-    url "https://github.com/lvandeve/lodepng.git",
-        revision: "48e5364ef48ec2408f44c727657ac1b6703185f8"
-  end
-
   resource "sjpeg" do
     url "https://github.com/webmproject/sjpeg.git",
         revision: "868ab558fad70fcbe8863ba4e85179eeb81cc840"
   end
 
   resource "skcms" do
     url "https://skia.googlesource.com/skcms.git",
-        revision: "64374756e03700d649f897dbd98c95e78c30c7da"
+        revision: "b25b07b4b07990811de121c0356155b2ba0f4318"
   end
 
   def install
     resources.each { |r| r.stage buildpath/"third_party"/r.name }
     # disable manpages due to problems with asciidoc 10
     system "cmake", "-S", ".", "-B", "build",
                     "-DJPEGXL_FORCE_SYSTEM_BROTLI=ON",
+                    "-DJPEGXL_FORCE_SYSTEM_LCMS2=ON",
```

Comment:
FWIW, libjxl would only use LCMS (`depends_on "little-cms2"`) when building with `-DJPEGXL_ENABLE_SKCMS=OFF`. If that's preferred, you can safely remove:
```ruby
  resource "skcms" do
    url "https://skia.googlesource.com/skcms.git",
        revision: "b25b07b4b07990811de121c0356155b2ba0f4318"
  end
```

Reply:
For reference, LCMS is used by Debian, while Fedora prefers skcms.
https://salsa.debian.org/debian-phototools-team/libjxl/-/blob/d5e9a5f19ead9dd70756fc0451a3309eee1cee08/debian/rules#L64
https://src.fedoraproject.org/rpms/jpegxl/blob/3b31ff8895fe7b15a7af6c89162fc96feaa71d48/f/jpegxl.spec#_79